docs/tables/aws_backup_vault.md

```diff
@@ -0,0 +1,33 @@
+# Table: aws_backup_vault
+
+AWS Backup vault is a container that you organize your backups in. You can use backup vaults to set the AWS Key Management Service (AWS KMS) encryption key that is used to encrypt backups in the backup vault and to control access to the backups in the backup vault.
+If you require different encryption keys or access policies for different groups of backups, you can optionally create multiple backup vaults. Otherwise, you can have all your backups organized in the default backup vault.
+
+## Examples
+
+### Basic Info
+
+```sql
+select
+  name,
+  arn,
+  creation_date
+from
+  aws_backup_vault;
+```
+
+
+### List vaults older than 90 days
+
+```sql
+select
+  name,
+  arn,
+  creation_date
+from
+  aws_backup_vault
+where
+  creation_date <= (current_date - interval '90' day)
+order by
+  creation_date;
+```
```

Comment:
@bigdatasourav can we add one more example such as 

`Ensure that the backups are encrypted at rest using KMS Customer Master Keys (CMKs).` 
Also we can do  

`Ensure that an Amazon Backup vault access policy is configured to prevent the deletion (accidentally or intentionally) of AWS backups in the backup vault. `

Reply:
doc updated for policy .